widepay.php

```diff
@@ -54,8 +54,11 @@ public function setMeta(array $meta = null)
     public function getSettings(array $meta = null)
     {
         // Load the view into this object, so helpers can be automatically add to the view
-        $this->view = new View('settings', 'default');
-        $this->view->setDefaultView('components' . DS . 'gateways' . DS . 'nonmerchant' . DS . 'widepay' . DS);
+        $this->view = $this->makeView(
+            'settings',
+            'default',
+            'components' . DS . 'gateways' . DS . 'nonmerchant' . DS . 'widepay' . DS
```

Comment:
I would upgrade the path in the third argument:

```
str_replace(ROOTWEBDIR, '', dirname(__FILE__) . DS)
```

Reply:
Done